### PR TITLE
[RSPS-1174] : Upgrade gh7

### PR DESCRIPTION
**Pull request by jp-lopez**
## Jira Ticket
[RSPS-1174](https://stuart-team.atlassian.net/browse/RSPS-1174)

## Implementation details
Merge GH 7. and some GH 8 changes until commit `3527752766daef864610d413532309c0519f5ceb`

Now is mandatory to define on the profiles the `ignored highways` section:

```
GraphHopperConfig config = new GraphHopperConfig();
        config.setProfiles(profiles);
        config.setCHProfiles(chProfiles);
        config.putObject("import.osm.ignored_highways","footway,cycleway,path,pedestrian,steps");
```

Adjust matrix profiles to follow the new GH format.

[RSPS-1174]: https://stuart-team.atlassian.net/browse/RSPS-1174?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by HDBandit — APPROVED** [review]
God bless the tests 🚀

**Review by tyoras — COMMENTED** [review]
I found one conflict that is not solved, luckily it was in the first files of the PR 😄

File: .github/workflows/publish-github-packages.yml [hunk 1]
```diff
@@ -31,9 +41,13 @@ jobs:
           restore-keys: |
             ${{ runner.os}}-node_modules-
       - name: Build and publish package ${{ matrix.java-version }}
+<<<<<<< HEAD:.github/workflows/push-packages.yml
+=======
+        # special 'wagon' option due to https://github.com/orgs/community/discussions/49001
+>>>>>>> 7.0:.github/workflows/publish-github-packages.yml
```
[the remark this answers is not in the document]
Comment (tyoras) [reply .github/workflows/publish-github-packages.yml:47]: I've made a search using my ide, this is the only one

**Review by tyoras — APPROVED** [review]
🚀

**Pull request merged**